www/m/js/layout/RadareInfiniteBlock.js

```diff
@@ -1,5 +1,5 @@
 import {NavigatorDirection} from '../core/NavigatorDirection';
-import {InfiniteScrolling} from '../helpers/InfiniteScrolling';
+import {InfiniteScrolling} from '../helpers/infiniteScrolling';
```

Comment:
We are importing the class from a file. The file name can be anything even ../helpers/abc
import {InfiniteScrolling} from '../helpers/abc';

Class Name is independent of Filename. Windows I think ignores case sensitivity cause @gcolajan had no issues in Windows, whereas I had when using Windows.

I'll rename the files to reflect their Internal Class Names so we don't have this issue again.